app/routes.py

```diff
@@ -1 +1,244 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, jsonify, abort, make_response, request
```

Comment:
👍🏾 These imports help a great deal when it comes to our request & response cycle